Homework-5/script.js

```diff
@@ -0,0 +1,148 @@
+// Task 1
+
+// let str = "Hi";
+// str.test = 5; // no,
+// console.log(str.test); // undefined
+
+// Task 2
+
+// getSum = () => {
+//   const sum = parseInt(prompt("Enter number 1")) + parseInt(prompt("Enter number 2"));
+//   alert(` Your sum is ${sum}`);
+// };
+
+// getSum();
+
+// Task 3
+
+// console.log((6.35).toFixed(1)); // loss of precision
+// console.log((6.35 * 10).toFixed(1) / 10);
+
+// Task 4
+
+// let i = 0;
+// while (i != 10) {
+//   i += 0.2;
+// } // infinite loop due to loss of precision: i == 9.999999999999996 next i == 10.199999999999996.
+
+// Task 5
+
+// const getRandom = (min, max) => {
+//     return min + Math.random() * (max - min);
+// }
+
+// console.log(getRandom(1, 5))
+
+// Task 6
+
+// const getRandomInteger = (min, max) => {
+//     return Math.round(min + Math.random() * (max - min));
+//     // return Math.floor(Math.random() * (max - min + 1)) + min; // found in learnjs
+
+// }
+
+// console.log(getRandomInteger(1, 5))
+
+// Task 7
+
+// const ucFirst = string => {
+//   if (string) {
+//     const result = string[0].toUpperCase() + string.slice(1);
+//     return result;
+//   } else {
+//     return string;
+//   }
+// };
```

Comment:
const ucFirst = str=> str ? str[0].toUpperCase() + str.slice(1) : str;
1. Use ternary operator
2. Try to avoid reserved words string => str, number => num